packages/backend/src/open_payments/payment_pointer/service.ts

```diff
@@ -110,7 +118,8 @@ async function getPaymentPointerByUrl(
 ): Promise<PaymentPointer | undefined> {
   const paymentPointer = await PaymentPointer.query(deps.knex)
     .findOne({ url })
-    .withGraphJoined('asset')
+    .withGraphFetched('asset')
+    .withGraphFetched('keys')
```

Comment:
do we also need this in the 
`src/open_payments/payment_pointer/service.ts` -> `processNextPaymentPointers` ?